Prithvi011-021/011-RecersiveFunctionToPrintPrimeNumber.js

```diff
@@ -0,0 +1,21 @@
+//Write a recursive function to print all the prime numbers between 1 to 100.
+// A prime number is a number 
+// which can either be divided by itself or 1
+
+let num = 100;
+for (let j = 2; j <= num; j++) {
+    let f = true;
+    // console.log(j);
+    for (let i = 2; i <= Number(j / 2); i++) {
```

Comment:
what is your reason to do j / 2 ?

Reply:
i took reference from google it j/2 is nothing but it will reduce looping through element if I took it will j-2 and also j - 3 it produced same output.